nextflow.config

```diff
@@ -152,7 +155,7 @@ profiles {
         }
         process {
             executor = 'slurm'
-            queue = 'componc'
+            queue = 'componc_cpu,componc_gpu'
         }
         executor {
             name = 'slurm'
```

Comment:
would it be a good idea to name this in a way for folks to understand which slurm cluster this intended for? not all slurms would have 'componc' queues

Reply:
my bad, it is already named `mskcc_iris`, thanks for this.